qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: Send CV
+
+  Scenario: positive sending CV
+  Given form "Send CV" with blank fields
+  When I enter valid data in the field "Name"
+  And I enter valid data in the field "Email"
+  And I attach CV
+  And I ticked "I agree to the personal data processing"
+  And I click button "Send"
+  Then my CV has been send
+
+
+  Scenario: exceeding the limit of characters in the field "Name"
```

Comment:
Also here you need to add the script number.

Reply:
done